tests/datadriven/test_variants.py

```diff
@@ -349,6 +341,44 @@ def convertPyvcfNumber(number):
             1 + len(self._formats) + len(self._infos) - gtCounter)
         self.assertEqual(len(keyMap), testMetaLength)
 
+    def testGetVariantsCallSets(self):
+        variantSet = self._gaObject
+        variantSetCompoundId = datamodel.VariantSetCompoundId.parse(
+            variantSet.getId())
+        start = 0
+        end = datamodel.PysamDatamodelMixin.vcfMax
+        callSetIds = variantSet.getCallSetIds()
+        someCallSetIds = callSetIds[0:3]
+        for referenceName in self._referenceNames:
+            # passing None as the callSetIds argument should be equivalent
+            # to passing all of the possible callSetIds as an argument
+            noneRecords = variantSet.getVariants(
+                referenceName, start, end, None, None)
+            allRecords = variantSet.getVariants(
+                referenceName, start, end, None, callSetIds)
```

Comment:
need to check that ``len(noneRecords) == len(allRecords)`` here (zip will run to the shortest of the two iterators).